server/graphql/v2/mutation/IndividualMutations.ts

```diff
@@ -93,6 +94,49 @@ const individualMutations = {
         });
       }
 
+      return { individual, token };
+    },
+  },
+  confirmEmail: {
+    description: 'Confirm email for Individual. Scope: "account".',
```

Comment:
Scope: "account" would imply it's oAuth but we're not checking scope. And do we really want oAuth tokens here?